integrations/resend/src/index.ts

```diff
@@ -33,13 +60,6 @@ export const sendEmail: AuthenticatedTask<
         },
         ...subjectProperty,
       ],
-      retry: {
```

Comment:
Don't remove this, these retry options will still be used in the `onError` callback returns undefined.